README.md

```diff
@@ -16,8 +16,13 @@ So far we tested Northstar on
 * 64-bit ARM
 * x86_64
 
-### Missing from implementation
+### Everything that is checked is implemented
```

Comment:
@flxo we should list also the implemented features from a birds eye view...please help me to complete this list

Reply:
@marcmo 

I'd add `chroot` and `Process supervision`